.babelrc.js

```diff
@@ -10,6 +11,7 @@ module.exports = {
     '@babel/plugin-proposal-class-properties',
     '@babel/plugin-proposal-object-rest-spread',
     '@babel/plugin-proposal-export-default-from',
+    ['@babel/plugin-proposal-decorators', { legacy: true }],
```

Comment:
I assume we need this only for angular, so let's have an override for angular with this (see the `overrides` down here)

Reply:
Vue.js with TS benefits from this as well https://vuejs.org/v2/guide/typescript.html#Class-Style-Vue-Components

Also, using decorators is a general TypeScript feature - does one babel plugin costs so much that it is worth to only use them for certain projects?

Reply:
Nah, just legacy: true makes me uncomfortable. But nvm then

Reply:
Check this link https://babeljs.io/docs/en/next/babel-plugin-proposal-decorators.html#legacy
I don't know what stage 1 or stage 2 is

I also don't know how decorators work in stage 2
If you think we should apply these only to vue and angular then lets do this.

But as said it does not work.. It compiles but brings up runtime errors in Storybook/Angular :(

Reply:
What are the runtime errors?

Reply:
Removed this line - for now, if we have a project that needs decorators, we use `tsc` instead of babel